backend/controller/SampleController.js

```diff
@@ -0,0 +1,58 @@
+const SampleModel = require('../model/SampleModel');
+
+const SampleController = {
+
+    async all (req, res) {
+        const data = await SampleModel.find();
+        res.send(data)
+    },
+
+    async byId (req, res) {
+        const data = await SampleModel.findById(req.params.id);
+        res.send(data)
+        
+    },
+
+    async create (req, res) {
+        const newSampleModel = new SampleModel({
+            title: req.body.title
+        });
+        await newSampleModel.save( (err, saved) => {
+            if (err) res.send(err);
+
+            res.send(saved)
+        })
+    },
+
+    async update (req, res) {
+        const data = await SampleModel.findByIdAndUpdate(
+            req.params.id,
+            { 
+                title: req.body.title,
+            },
+            { new: true }
+          );
+
+          if (!data){
+            return res
+              .status(404)
+              .send("Data with id not found");
+          }
+        
+          res.send(data);
+    },
+
+    async remove (req, res) {
```

Comment:
don't use async functions please convert them to normal function with the .then() and .catch()

Reply:
Sure, will do that

Reply:
make sure your code passes all the CI tests